### DISCUSSION TITLE
How to Report issue with Developmental/Experimental Build to Developers

### DISCUSSION THREAD

**Discussion by EmbeddedInvented** [blakeblackshear/frigate#10600] [General]
I know I "promised" I wouldn't make an issue on Github when running the developmental builds however I would like to contribute in some way even though I am unable to submit a PR myself due to unfamiliarity with typescript/react. 

Is there any "accepted" way to report bugs on these builds? 

I recently noticed an issue where preview thumbnails were not loading in the motion events page for cameras with a "-" in the name specifically for the current hour time period. This seems to be a bug introduced with  #10384, 

my intuition would be it is in web/src/components/player/PreviewPlayer.tsx 

with the code 
`      parseFloat(frame.split("-")[1].slice(undefined, -5)),` 

However again I am not certain this is exactly the root cause, just want to try and help in any way I can!

**Comment by xfaustus** [comment]
Same issue for me. I think would be good idea to start reporting issue while proceding with 0.14 to avoid to accumulate too many bugs.
Thanks

**Comment by NickM-27** [comment]
The request to not report anything exists because things are still changing and not final. We don't want to have to respond to or explain to bug reports for things that we are already planning on redoing or changing in a larger way. 

It will still be multiple months before there is feature parity from the old webui (debug, system page, zone / mask editor, etc)

There will be plenty of time for bug reports and user feedback once we get closer and the core functionality is finalized
